src/ProjectTemplates/Web.ProjectTemplates/content/WebApi-CSharp/Controllers/ValuesController.cs

```diff
@@ -1,60 +0,0 @@
-﻿using System;
```

Comment:
You'll need to [update the baselines](https://github.com/aspnet/AspNetCore/blob/master/src/ProjectTemplates/test/template-baselines.json#L398) to reflect the new file names.